runtime/runtime/src/config.rs

```diff
@@ -172,7 +172,7 @@ pub fn tx_cost(
     let maximum_depth =
         if minimum_new_receipt_gas > 0 { prepaid_gas / minimum_new_receipt_gas } else { 0 };
     let inflation_exponent =
-        u8::try_from(initial_receipt_hop + maximum_depth).map_err(|_| IntegerOverflowError {})?;
+        u8::try_from(initial_receipt_hop + maximum_depth).unwrap_or_else(|_| u8::max_value());
```

Comment:
@evgenykuzyakov should we cap it at 63 instead?

Reply:
Why do we use this instead of setting `minimum_new_receipt_gas` to `0`?

Reply:
@evgenykuzyakov we don't want to change the fees (in gas) for the purpose of parameter estimator.

Reply:
If `minimum_new_receipt_gas` is `0`, then the `maximum_depth` is 0, so it doesn't overflow.